sbt-scalajs-reflection/src/main/scala/be/doeraene/sjsreflect/sbtplugin/ReflectSelectors.scala

```diff
@@ -27,6 +27,11 @@ object ReflectSelectors {
   /** Allows to find the selected by its fully qualified name. */
   private[sbtplugin] case object ReflectClassByName extends Operation
 
+  /**
+    * Allows classes selected by this modifier to be enumerated as an array
+    */
+  private[sbtplugin] case object ReflectEnumerateClass extends Operation
```

Comment:
Please follow the style of comments. Either right above if single-line, or right below if multi-line.